src/components/calcite-flow-item/calcite-flow-item.e2e.ts

```diff
@@ -67,6 +67,16 @@ describe("calcite-flow-item", () => {
     expect(element).toEqualText("test");
   });
 
+  it("should have default summary", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<calcite-flow-item summary="test"></calcite-flow-item>');
+
+    const element = await page.find(`calcite-flow-item >>> .${CSS.summary}`);
+
+    expect(element).toEqualText("test");
```

Comment:
Suggestion to move the string to a TEXT content in resources, so updates to components keep the tests in sync.

Reply:
Its local to this file so it wouldn't get out of sync.